### DISCUSSION TITLE
Debug with breakpoints in webstorm

### DISCUSSION THREAD

**Discussion by huang-julien** [nuxt/nuxt#17222] [Questions]
Hi i'm currently working with webstorm and the only way to stop the js execution at a specific point in the code is to set the statement "debugger". I set some breakpoints but it seems that it just doesn't recognize any of it. I usually start the inspector with --inpect-brk. Is there any additionnal configuration to set on webstorm ? I also have the same issue when working with nitro

**Comment by Harm-Nullix** [comment]
Linked issue: https://github.com/nuxt/nuxt/issues/13875

This is a real deal breaker. 
You need to be able to debug your nodejs server properly using a nodejs debugger
